src/extensions/yfm/YfmNote/commands.ts

```diff
@@ -31,6 +31,37 @@ export const exitFromNoteTitle: Command = (state, dispatch) => {
     return true;
 };
 
+export const liftEmptyBlockFromNote: Command = (state, dispatch) => {
+    const {selection, schema} = state;
+    if (!isTextSelection(selection)) return false;
+
+    const {$cursor} = selection;
+    // cursor should be inside an empty textblock
+    if (!$cursor || $cursor.parent.content.size) return false;
+
+    if (
+        $cursor.depth > 2 && // depth must be at least 3
+        isSameNodeType($cursor.node(-1), noteContentType(schema)) &&
+        isSameNodeType($cursor.node(-2), noteType(schema)) &&
+        $cursor.node(-1).childCount > 1
+    ) {
+        // current texblock is last child
+        if ($cursor.after() === $cursor.end(-1)) {
+            const nodeRange = new NodeRange(
+                state.doc.resolve($cursor.before()),
+                state.doc.resolve($cursor.after()),
+                $cursor.depth - 1,
+            );
```

Comment:
Can we replace this with `$cursor.blockRange()!` like in YfmCut?

Reply:
fixed